<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1250</width>
    <height>800</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>微软雅黑</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QSplitter" name="m_splitter_x">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="handleWidth">
       <number>5</number>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QComboBox" name="m_ComBox_HugeSectionName">
          <property name="maxVisibleItems">
           <number>35</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="m_TabVw_HugeSectionData"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="verticalLayoutWidget_2">
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="95,5">
        <item>
         <widget class="QSplitter" name="m_splitter_z">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="verticalLayoutWidget_3">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTabWidget" name="m_TabWid">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <widget class="QWidget" name="tab">
               <attribute name="title">
                <string>过滤搜索</string>
               </attribute>
               <layout class="QGridLayout" name="m_GridLayout_Tab_1" columnstretch="5,20,5,20,5,35,5,5">
                <property name="sizeConstraint">
                 <enum>QLayout::SetMinAndMaxSize</enum>
                </property>
                <property name="leftMargin">
                 <number>5</number>
                </property>
                <property name="topMargin">
                 <number>5</number>
                </property>
                <property name="rightMargin">
                 <number>5</number>
                </property>
                <property name="bottomMargin">
                 <number>5</number>
                </property>
                <property name="spacing">
                 <number>4</number>
                </property>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>字段类型：</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2" colspan="4">
                 <widget class="QLineEdit" name="m_Edit_Search_Contents"/>
                </item>
                <item row="1" column="2">
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>条件：</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="4">
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>数值：</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="3">
                 <widget class="QComboBox" name="m_ComBo_Filter_Condition">
                  <item>
                   <property name="text">
                    <string>等于</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>大于</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>小于</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>大于等于</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>小于等于</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>包含</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="0" column="6">
                 <widget class="QPushButton" name="m_But_Search">
                  <property name="text">
                   <string>搜索</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="m_ComBo_Search_Type">
                  <item>
                   <property name="text">
                    <string>按&lt;ID&gt;搜索</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>按&lt;名称&gt;搜索</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>搜索类型：</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="m_ComBo_Filter_Type"/>
                </item>
                <item row="1" column="5">
                 <widget class="QLineEdit" name="m_Edit_Filter_Contents"/>
                </item>
                <item row="1" column="6">
                 <widget class="QPushButton" name="m_But_Filter">
                  <property name="text">
                   <string>过滤</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="7" rowspan="2">
                 <widget class="QPushButton" name="m_But_Search_Reset">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>16777215</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>重置搜索/过滤</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
              <widget class="QWidget" name="tab_2">
               <attribute name="title">
                <string>全局搜索</string>
               </attribute>
              </widget>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QScrollArea" name="scrollArea">
           <property name="widgetResizable">
            <bool>false</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>392</width>
              <height>69</height>
             </rect>
            </property>
           </widget>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,2,2,2,5,70,16">
          <item>
           <widget class="QPushButton" name="m_But_UpPage">
            <property name="text">
             <string>上一页</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="m_Label_MinPage">
            <property name="text">
             <string>1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>/</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="m_Label_MaxPage">
            <property name="text">
             <string>1</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="m_But_NextPage">
            <property name="text">
             <string>下一页</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton">
            <property name="text">
             <string>保存当前页面</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="m_ToolBar_Main">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonIconOnly</enum>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_Open"/>
   <addaction name="action_Save"/>
   <addaction name="separator"/>
   <addaction name="action_Ver"/>
   <addaction name="action_AboutMe"/>
  </widget>
  <action name="action_Open">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/build/Debug/interface/icon/ToolBar_open.png</normaloff>:/build/Debug/interface/icon/ToolBar_open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
   <property name="toolTip">
    <string>Open_Elements.data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/build/Debug/interface/icon/ToolBar_Save.png</normaloff>:/build/Debug/interface/icon/ToolBar_Save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>Save_Elements.data</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="action_Ver">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/build/Debug/interface/icon/ToolBar_Ver.png</normaloff>:/build/Debug/interface/icon/ToolBar_Ver.png</iconset>
   </property>
   <property name="text">
    <string>版本信息</string>
   </property>
   <property name="toolTip">
    <string>Versions</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WindowShortcut</enum>
   </property>
   <property name="priority">
    <enum>QAction::NormalPriority</enum>
   </property>
  </action>
  <action name="action_AboutMe">
   <property name="icon">
    <iconset resource="mainwindow.qrc">
     <normaloff>:/build/Debug/interface/icon/ToolBar_InRegardTo.png</normaloff>:/build/Debug/interface/icon/ToolBar_InRegardTo.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
   <property name="toolTip">
    <string>AboutMe</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
